Order of CMake add_subdirectory for remote services

I found that [0c64ca6a1a97b6f997d1d7773d1c53c79c102bbd](https://github.com/apache/celix/commit/0c64ca6a1a97b6f997d1d7773d1c53c79c102bbd) introduced the following CMake error:
```bash
cmake -DCMAKE_BUILD_TYPE:STRING=Debug -DCMAKE_EXPORT_COMPILE_COMMANDS:BOOL=ON -DENABLE_TESTING:BOOL=FALSE -DENABLE_ADDRESS_SANITIZER:BOOL=ON -DBUILD_RSA_DISCOVERY_ETCD:BOOL=ON -DBUILD_RSA_REMOTE_SERVICE_ADMIN_DFI:BOOL=ON -G "Unix Makefiles" /home/peng/Downloads/git/mycelix 
-- Using jar to create bundles

CMake Error at cmake/cmake_celix/ContainerPackaging.cmake:424 (message):
  Cannot add bundle in container remote-services-dfi.  Provided bundle is not
  a abs path to an existing file nor a cmake target
  (Celix::rsa_discovery_etcd).
Call Stack (most recent call first):
  cmake/cmake_celix/ContainerPackaging.cmake:528 (celix_container_bundles_dir)
  cmake/cmake_celix/ContainerPackaging.cmake:327 (celix_container_bundles)
  bundles/remote_services/examples/CMakeLists.txt:52 (add_celix_container)


-- Configuring incomplete, errors occurred!
See also "/home/peng/Downloads/git/mycelix/build/Debug/CMakeFiles/CMakeOutput.log".
```

Should `add_subdirectory(examples)` be moved to its original place?

--- a/bundles/remote_services/CMakeLists.txt
+++ b/bundles/remote_services/CMakeLists.txt
@@ -42,6 +42,39 @@ if (REMOTE_SERVICE_ADMIN)
         #add_subdirectory(remote_service_admin_shm)
     endif ()
 
+    if (BUILD_RSA_DISCOVERY_ETCD AND BUILD_RSA_REMOTE_SERVICE_ADMIN_DFI)
+        add_celix_container(remote-services-dfi
+                NAME "server"
+                GROUP "remote-services/remote-services-dfi"
+                BUNDLES
+                Celix::rsa_discovery_etcd
+                Celix::rsa_topology_manager
+                Celix::rsa_dfi
+                Celix::shell
+                Celix::shell_tui
+                Celix::log_admin
+
+                celix_remote_interceptors_example
+                calculator
+                PROPERTIES
+                RSA_PORT=18888
+                )
+
+        add_celix_container(remote-services-dfi-client
+                NAME "client"
+                GROUP "remote-services/remote-services-dfi"
+                BUNDLES
+                Celix::rsa_topology_manager
+                Celix::rsa_dfi Celix::shell
+                Celix::shell_tui
+                Celix::log_admin
+                Celix::rsa_discovery_etcd
+                celix_remote_interceptors_example
+                calculator_shell
+                PROPERTIES
+                RSA_PORT=28888
+                )
+    endif()
 endif (REMOTE_SERVICE_ADMIN)
 
 

--- a/bundles/remote_services/examples/CMakeLists.txt
+++ b/bundles/remote_services/examples/CMakeLists.txt
@@ -46,38 +46,3 @@ add_subdirectory(interceptors)
 #            BUNDLES org.apache.celix.calc.api.Calculator_proxy
 #        )
 #    endif ()
-
-
-if (BUILD_RSA_DISCOVERY_ETCD AND BUILD_RSA_REMOTE_SERVICE_ADMIN_DFI)
-    add_celix_container(remote-services-dfi
-        NAME "server"
-        GROUP "remote-services/remote-services-dfi"
-        BUNDLES
-            Celix::rsa_discovery_etcd
-            Celix::rsa_topology_manager
-            Celix::rsa_dfi
-            Celix::shell
-            Celix::shell_tui
-            Celix::log_admin
-
-            celix_remote_interceptors_example
-            calculator
-        PROPERTIES
-            RSA_PORT=18888
-    )
-
-    add_celix_container(remote-services-dfi-client
-        NAME "client"
-        GROUP "remote-services/remote-services-dfi"
-        BUNDLES
-            Celix::rsa_topology_manager
-            Celix::rsa_dfi Celix::shell
-            Celix::shell_tui
-            Celix::log_admin
-            Celix::rsa_discovery_etcd
-            celix_remote_interceptors_example
-            calculator_shell
-        PROPERTIES
-            RSA_PORT=28888
-    )
-endif ()

